connectors/uplynk/src/main/java/com/theoplayer/android/connector/uplynk/UplynkListener.kt

```diff
@@ -18,6 +20,15 @@ interface UplynkListener {
      */
     fun onPreplayResponse(response: PreplayResponse) {}
 
+    /**
+     * Called when a preplay response is received from Uplynk for live channel or an event.
+     *
+     * For more details, refer to the [Preplay API (Version 2) Documentation](https://docs.edgecast.com/video/index.html#Develop/Preplayv2.htm).
+     *
+     * @param response the `PreplayLiveResponse` object containing information relevant to the preplay request.
+     */
+    fun onPreplayResponseLive(response: PreplayLiveResponse){}
```

Comment:
Probably better to keep the method name in sync with the class name: 
```suggestion
    fun onPreplayLiveResponse(response: PreplayLiveResponse){}
```